projects/plugins/social/src/js/components/social-module-toggle/index.tsx

```diff
@@ -46,18 +43,8 @@ const SocialModuleToggle: React.FC = () => {
 					{ __( 'Learn more', 'jetpack-social' ) }
 				</ExternalLink>
 			</Text>
-			{ connectionsAdminUrl && (
-				<Button
-					fullWidth={ isSmall }
-					className={ styles.button }
-					variant="secondary"
-					isExternalLink={ true }
-					href={ connectionsAdminUrl }
-					disabled={ isUpdating || ! isModuleEnabled }
-					target="_blank"
-				>
-					{ __( 'Manage social media connections', 'jetpack-social' ) }
-				</Button>
+			{ ! isUpdating && isModuleEnabled && (
+				<ConnectionManagement className={ styles[ 'connection-management' ] } />
```

Comment:
Should we retain the existing UI based on the feature flag?

Reply:
Yes good point, I was AFK yesterday when the feature flag got merged, but rebasing it now 👍